package.json

```diff
@@ -56,18 +55,19 @@
 		"build-docker": "node bin/build-docker.js",
 		"build-static": "npx ncp static public",
 		"prebuild-server": "mkdirp build",
-		"build-server": "cross-env-shell BROWSERSLIST_ENV=server NODE_PATH=$NODE_PATH:server:client:. webpack --display errors-only --config client/webpack.config.node.js",
+		"build-server": "cross-env-shell BROWSERSLIST_ENV=server webpack --display errors-only --config client/webpack.config.node.js",
 		"build-client": "npm run build-client-evergreen",
-		"build-client-fallback": "cross-env-shell BROWSERSLIST_ENV=defaults NODE_PATH=$NODE_PATH:server:client:. node $NODE_ARGS ./node_modules/webpack/bin/webpack.js --config client/webpack.config.js --display errors-only && npm run validate-fallback-es5",
-		"build-client-evergreen": "cross-env-shell BROWSERSLIST_ENV=evergreen NODE_PATH=$NODE_PATH:server:client:. node $NODE_ARGS ./node_modules/webpack/bin/webpack.js --config client/webpack.config.js --display errors-only",
+		"build-client-fallback": "cross-env-shell BROWSERSLIST_ENV=defaults node $NODE_ARGS ./node_modules/webpack/bin/webpack.js --config client/webpack.config.js --display errors-only",
+		"postbuild-client-fallback": "npm run validate-fallback-es5",
+		"build-client-evergreen": "cross-env-shell BROWSERSLIST_ENV=evergreen node $NODE_ARGS ./node_modules/webpack/bin/webpack.js --config client/webpack.config.js --display errors-only",
 		"build-client-both": "CONCURRENT_BUILDS=2 concurrently -c cyan -n \"fallback ,evergreen\" \"npm run build-client-fallback\" \"npm run build-client-evergreen\"",
 		"build-client-if-prod": "node -e \"process.env.CALYPSO_ENV === 'production' && process.exit(1)\" || cross-env-shell NODE_ARGS=${NODE_ARGS:---max_old_space_size=8192} npm run build-client-both",
 		"build-client-if-desktop": "node -e \"(process.env.CALYPSO_ENV === 'desktop' || process.env.CALYPSO_ENV === 'desktop-development') && process.exit(1)\" || cross-env-shell NODE_ARGS=${NODE_ARGS:---max_old_space_size=8192} npm run build-client-fallback",
 		"build-packages": "npx lerna run prepare --stream",
 		"build-packages:watch": "chokidar \"./packages/**\" -i \"./packages/*/dist/**\" -i \"**.css\" -i \"**.d.ts\" --debounce 1000 --throttle 10000 -c \"npm run build-packages\"",
-		"clean": "npm run -s clean:build && npm run -s clean:devdocs && npm run -s clean:public && npm run -s clean:packages",
-		"clean:build": "npx rimraf build server/bundler/*.json .babel-cache",
-		"clean:devdocs": "npx rimraf server/devdocs/search-index.js",
+		"clean": "run-s -s clean:*",
```

Comment:
Very nice 👍

Reply:
`run-p` might be even faster for those of us with SSDs

Reply:
I considered `run-p`, too, but was afraid of race conditions like `npx rimraf` doing the download and install two times in parallel.